packages/wonder-blocks-button/src/components/button-core.tsx

```diff
@@ -84,12 +84,12 @@ const ButtonCore: React.ForwardRefExoticComponent<
             kind !== "tertiary" &&
                 !disabled &&
                 (pressed
-                    ? buttonStyles.active
-                    : (hovered || focused) && buttonStyles.focus),
+                    ? buttonStyles.pressed
+                    : focused && buttonStyles.focused),
```

Comment:
**note:** Changed the style names to match what's returned from `clickable-behavior`.